### PR TITLE
Testing slack integration

### PR DESCRIPTION
**Pull request by stilesj-uchicago**
### Improvements
Testing out slack PR notifications.

### CODE REVIEW

**Review by mpsolano — CHANGES_REQUESTED** [review]
I have reviewed and request changes.

**Comment by stilesj-uchicago** [comment]
> I have reviewed and request changes.

Thank you!

File: README.slack.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Github slack integration: https://github.com/integrations/slack
+1. Add the App to Slack
+1. Login `/github signin`
+1. Register a repository
```
Comment (mpsolano) [inline README.slack.md:4]: This

**Review by mpsolano — COMMENTED** [review]
Testing comment without approval

**Review by mpsolano — APPROVED** [review]
Testing approval

**Comment by stilesj-uchicago** [comment]
Comments should be disabled now.

File: README.slack.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	```
+	/github subscribe NCI-GDC/indexclient pulls reviews comments
```
Comment (mpsolano) [inline README.slack.md:6]: This too

**Review by mpsolano — COMMENTED** [review]
Added another comment

File: README.slack.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Github slack integration: https://github.com/integrations/slack
```
Comment (mpsolano) [inline README.slack.md:1]: And this

**Review by mpsolano — COMMENTED** [review]
Commented without approval

**Pull request closed without merging**